### PR TITLE
ticket 6

### PR DESCRIPTION
**Pull request by Stephen-I**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -100,4 +100,29 @@ describe("/api/app", () => {
...
+        });
+      });
+  });
+  test("Search for Comments using review_id with properties of body, review_id, votes, comment_id, created_at, author", () => {
+    return request(app)
+      .get("/api/reviews/5/comments")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.comments.length).toBe(0);
+      });
```
Comment (pippaamy) [inline __tests__/app.test.js:125]: Lets add some tests to check for:
Status 400, invalid ID, 
Status 404, non existent ID, e.g 9999

**Review by pippaamy — COMMENTED** [review]
Good work, lets add those extra comments :)

File: __tests__/app.test.js [hunk 2]
```diff
@@ -100,4 +100,46 @@ describe("/api/app", () => {
...
+          'invalid input syntax for type integer: "notAnID"'
+        );
+      });
+  });
+  test("status:404, responds with an error message when passed unavailable ID", () => {
+    return request(app)
+      .get("/api/reviews/10000/comments")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("No reviews found for review_id: 10000");
```
Comment (Pipe-Smoking-Rabbit) [inline __tests__/app.test.js:142]: very cool to actually get the specific review ID in the message 😎

File: models/comment.model.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+        });
+      }
+
+      return db.query("SELECT * FROM comments WHERE review_id = $1", [
+        review_id,
+      ]);
+    })
+    .then(({ rows }) => {
+      const comment = rows;
+      return comment;
```
Comment (Pipe-Smoking-Rabbit) [inline models/comment.model.js:22]: Really nice handling of this very tricky double-query in the model 👏

**Review by Pipe-Smoking-Rabbit — APPROVED** [review]
Ticket 6 complete ✅

**Pull request merged**